src/User/Infrastructure/Repository/UserRepository.php

```diff
@@ -14,14 +15,23 @@ public function __construct(EntityManagerInterface $entityManager)
         $this->entityManager = $entityManager;
     }
 
+    public function delete(User $user): void
+    {
+        $deleteTodos = new TodoRepository($this->entityManager);
+        $deleteTodos->deleteAllByUserId($user->id);
```

Comment:
options to improve:
1. handle this by the usecase
2. "createQuery" to handle this without calling the repository
3. use aggregates